src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -1614,7 +1614,7 @@ JFR_ONLY(int java_lang_Thread::_jfr_epoch_offset;)
 #define THREAD_FIELDS_DO(macro) \
   macro(_holder_offset,        k, "holder", thread_fieldholder_signature, false); \
   macro(_name_offset,          k, vmSymbols::name_name(), string_signature, false); \
-  macro(_contextClassLoader_offset, k, vmSymbols::contextClassLoader_name(), classloader_signature, false); \
+  macro(_contextClassLoader_offset, k, "contextClassLoader", classloader_signature, false); \
```

Comment:
I didn't think the context class loader was related to SM in any way. ??

Reply:
It isn't.  This symbol was near the ones I deleted, and I deleted it by mistake, so I moved it here.